test/functional/api/v1/smart_proxies_controller_test.rb

```diff
@@ -47,7 +47,7 @@ class Api::V1::SmartProxiesControllerTest < ActionController::TestCase
   end
 
   test "should update smart_proxy" do
-    put :update, { :id => smart_proxies(:one).to_param, :smart_proxy => { } }
+    put :update, { :id => smart_proxies(:one).to_param, :smart_proxy => valid_attrs }
```

Comment:
Updating any object using a payload of `{}` will trigger strong_params as it doesn't contain permitted attributes and will throw an error.